bridges/snowbridge/test-utils/src/mock_inbound_queue.rs

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+
+use snowbridge_core::reward::{AddTip, AddTipError};
+
+pub struct MockOkInboundQueue;
+
+impl AddTip for MockOkInboundQueue {
+	fn add_tip(nonce: u64, _amount: u128) -> Result<(), AddTipError> {
+		// Force an error condition
+		if nonce == 3 {
```

Comment:
You could put the `3` into a constant like `FAILING_NONCE` so it's clearer

Reply:
Addressed in https://github.com/paritytech/polkadot-sdk/pull/8271/commits/78543ef342812e723200da5655a3f9f514644171.